os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -71,11 +71,16 @@ config AMEBASMART_SPI
 	default n
 	
 if AMEBASMART_SPI
+config RTL8730E_SPI0
+	bool "Enable SPI0"
+	default n
+config RTL8730E_SPI1
```

Comment:
When do you use `AMEBASMART` or `RTL8730E`?

Reply:
I modified the condition now to AMEBASMART to match with other peripherals